test/test_ui_state.js

```diff
@@ -841,6 +841,24 @@ describe('UI State object', function() {
       });
     });
 
+    it('parses urls with a baseUrl', function() {
+      var saveStub = testUtils.makeStubMethod(state, 'saveState');
+      this._cleanups.push(saveStub.reset);
+      var oldBaseUrl = state.get('baseUrl');
```

Comment:
A new state instance is generated for every test so this shouldn't be necessary. If it is could you please investigate further.